app/templates/main/search-transferring-body.html

```diff
@@ -90,42 +90,47 @@ <h2 class="govuk-body browse__body">You are viewing</h2>
+                                                            </td>
+                                                            <td class="govuk-table__cell govuk-table--invisible-on-mobile">
+                                                                <a href="{{ url_for('main.browse_consignment', _id=record['_source']['consignment_id']) }}">{{ record['_source']["consignment_reference"] }}</a>
+                                                            </td>
+                                                            <td class="govuk-table__cell govuk-table--invisible-on-mobile">
+                                                                <a class="word-break"
+                                                                   href="{{ url_for('main.record', record_id=result['file_id']) }}">{{ result["file_name"] }}</a>
+                                                            </td>
+                                                            <td class="govuk-table__cell">
+                                                                <strong class="{% if result['closure_type'] == 'Open' %}govuk-tag govuk-tag--green{% elif result['closure_type'] == 'Closed' %}govuk-tag govuk-tag--red{% elif result['closure_type'] is none %}{% endif %}">
+                                                                    {{ result['closure_type'] }}
+                                                                </strong>
+                                                            </td>
+                                                            <td class="govuk-table__cell govuk-table--right-align">
+                                                                {% if result['opening_date'] %}
+                                                                    {{ result['opening_date'] }}
+                                                                {% else %}
+                                                                    –
+                                                                {% endif %}
+                                                            </td>
+                                                        </tr>
+                                                    {% endif %}
+                                                {% endfor %}
+                                            {% endif %}
                                         </div>
                                     </tbody>
                                 </table>
                                 <!-- PAGINATION -->
                                 {% with view_name='main.search_transferring_body', id = request.view_args['_id'] %}
-                                    {% include "pagination.html" %}
+                                    {% include "pagination-new.html" %}
```

Comment:
can we use the new pagination for all the routes and justget rid of the old one? It can wait for a separate PR but just checking that the logic should be universal right?